enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -1434,10 +1472,11 @@ func (p *PebbleCache) FindMissing(ctx context.Context, resources []*rspb.Resourc
 		}
 
 		unlockFn := p.locker.RLock(key.LockID())
-		found, err := p.iterHasKey(iter, key)
-		if !found || err != nil {
+		lastAccessUsec, err := p.lookupLastAccessTime(iter, key)
+		if err != nil {
 			missing = append(missing, r.GetDigest())
 		}
+		p.sendAtimeUpdate(key, lastAccessUsec)
```

Comment:
I don't think we need to sendAtimeUpdate if the file was missing. (correct me if i'm wrong?)